docs/admin_guide.md

```diff
@@ -264,7 +264,7 @@ REDIS_DNS=redis://${REDIS_PASSWORD}@${REDIS_HOST}
 
 ### PostgreSQL (Database)
 
-Create new `kbin` database user, using the password, `{!SECRET!!KEY!-32_2-!}`, you generated earlier:
+Create a new `kbin` database user, using the password, `{!SECRET!!KEY!-32_2-!}`, you generated earlier:
```

Comment:
This fix is correct, but the sentence is still awkward and needs more work.
I would suggest something like this:
Create a new `kbin` database user, using the password you generated earlier (`{!SECRET!!KEY!-32_2-!}`):